packages/playground/src/components/image-carousel.tsx

```diff
@@ -20,16 +20,9 @@ interface ImageCarouselProps {
 // TODO: rewrite this to actually work
 const ImageCarousel = (props: ImageCarouselProps) => {
   const { isOpen, setIsOpen, setZoomLevel, zoomLevel, imgEvent } = props
-  const [
-    imgTitle
-    // setImageTitle
-  ] = useState(imgEvent[0])
+  const [imgTitle] = useState(imgEvent[0])
```

Comment:
Does it need a null check here `imgEvent?.[0]`?

Reply:
added null check